### PR TITLE
Release only once on travis

### PR DESCRIPTION
**Pull request by KevinEtienne**
Add a variable to check when to deploy

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ env:
     - TOX_ENV=py33-django16
     - TOX_ENV=py33-django17
     - TOX_ENV=py34-django16
-    - TOX_ENV=py34-django17
+    - TOX_ENV=py34-django17 DEPLOY_RELEASE=true
```
Comment (meshy) [inline .travis.yml]: What if the other builds fail?

**Comment by KevinEtienne** [comment]
@Ian-Foote @meshy this might allow us to create a release only once, still looking if I can find other solution but otherwise feel free to comment :)

File: .travis.yml (hunk 1 above)
Comment (KevinEtienne) [inline .travis.yml]: I guess it will never be released, are we likely to allow to release a tag if the build fail?

Comment (Ian-Foote) [inline .travis.yml]: I don't think we care: we should already have passing tests on master by the time we push a tag. I'd prefer the deploy was completely distinct from the tests, but this is an improvement in my opinion.

**Comment by KevinEtienne** [comment]
Feel free to merge if we don't have another solution

**Comment by Ian-Foote** [comment]
Looks good to me. We can update again later if we think of something better.

**Pull request merged**